cypresse2e/cypress/integration/e2eTests/feedbackModal/feedbackModal.spec.js

```diff
@@ -0,0 +1,81 @@
+import FeedbackModal from '../../../elements/pages/feedbackModal';
+
+describe('FightPandemics Feedback Modal', () => {
+
+    const feedback = new FeedbackModal();
+
+    context('User provides feedback on FightPandemics', () => {
+        before(() => {
+            feedback.visit();
+        });
+
+        it('Feedback icon on homepage is visible and clickable', () => {
+            feedback.getFeedbackIcon()
+                .should('be.visible').click();
+        });
+
+        it('Feedback scale heading is visible', () => {
+            feedback.getFeedbackHeadingOne()
+                .should('be.visible')
+                .contains('How well does FightPandemics meet your needs?');
+        });
+
+        it('Feedback scale text is visible', () => {
+            feedback.getFeedbackScaleText()
+                .should('be.visible')
+                .contains('Poorly' + 'Very well');
+        });
+
+        context('User chooses a rating on the scale', () => {
+            beforeEach(() => {
+                feedback.visit();
+                feedback.getFeedbackIcon()
+                    .should('be.visible').click();
+            });
+
+            it('Choosing rating 1', () => {
+                clickRating().eq(0)
+            });
+            it('Choosing rating 2', () => {
+                feedback.getFeedbackScale().find('div').eq(1).click();
+            });
+            it('Choosing rating 3', () => {
+                feedback.getFeedbackScale().find('div').eq(2).click();
+            });
+            it('Choosing rating 4', () => {
+                feedback.getFeedbackScale().find('div').eq(3).click();
+            });
+            it('Choosing rating 5', () => {
+                feedback.getFeedbackScale().find('div').eq(4).click();
+            });
+
+        
+            // it.skip('Feedback scale numbers are visible and clickable', () => {
```

Comment:
Hi @jessica-lau 
Are we going to use this commented out code? Or we can remove it?

Reply:
Yes, I'll remove it when I'm finish with writing the test. 👍